garageScript/c0d3-app
# Add an Info Alert when there's nothing to review

Closes #2462 

### Description

Add an info to inform the user that there's nothing to review at the moment.

<details>
<summary>
Image
</summary>
<img width="1357" alt="image" src="https://user-images.githubusercontent.com/35906419/200118962-580b2653-39d4-4dcd-9d11-3daa9d06edef.png">
</details>

### Testing

1. Go to `/review/js1`
2. Make sure it displays `All the submissions were reviewed. Come and check later!`
3. Go to `/review/js0`
4. Make sure it shows the submissions
